src/main/java/seedu/address/model/module/Credit.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.module;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Module's credits in the transcript.
+ * <p>
+ * Guarantees: immutable; is valid as declared in {@link #isValidCredit(int)}
+ */
+public class Credit {
```

Comment:
Similarly to this, Credit class was not initially in the class diagram.

Reply:
Yep, going to update documentations for both lucidchart and the developers guide once i'm done adding test.